src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -3,7 +3,24 @@
 
 public class Task10 {
 
-    public static boolean containDigitTwo(int n) {
-        return false;
+    public static boolean containDigitTwo(int n)
```

Comment:
It can be much simpler. Think about '%' operator and how it can be applied there.